/*
*   Copyright 2020 Google LLC
*
*   Licensed under the Apache License, Version 2.0 (the "License");
*   you may not use this file except in compliance with the License.
*   You may obtain a copy of the License at
*
*       https://www.apache.org/licenses/LICENSE-2.0
*
*   Unless required by applicable law or agreed to in writing, software
*   distributed under the License is distributed on an "AS IS" BASIS,
*   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
*   See the License for the specific language governing permissions and
*   limitations under the License.
*/

syntax = "proto2";

package wireless_android_play_analytics;

import "fetcher/proto/enums.proto";

// Next tag: 3
message HologramConfig {
    optional DataType source_type = 1;
    optional Corpus kvick_corpus = 2;
}

// Next tag: 2
message HologramConfigSet {
    repeated HologramConfig configs = 1;
}